src/net.cpp

```diff
@@ -937,6 +940,16 @@ bool CConnman::AttemptToEvictConnection()
     return false;
 }
 
+int CConnman::CountInboundConnections() const
+{
+    int inbounds = 0;
+    LOCK(cs_vNodes);
+    for (const CNode* node : vNodes) {
```

Comment:
Can we count that such connections are from non-local addresses? Should be an extra two LoC